internal/mithril-persistence/src/database/repository/cardano_transaction_repository.rs

```diff
@@ -90,8 +91,26 @@ impl CardanoTransactionRepository {
         let records: Vec<CardanoTransactionRecord> =
             transactions.into_iter().map(|tx| tx.into()).collect();
 
```

Comment:
You can remove this transformation since `create_transactions_with_connection` will do it for you.